packages/sshnoports/bundles/shell/headless/sshnpd.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+sleep 10; # allow machine to bring up network
+while true; do
+  # disable "var is referenced but not assigned" warning for template
+  # shellcheck disable=SC2154
+  "$HOME"/.local/bin/sshnpd -a "$device_atsign" -m "$manager_atsign" -d "$device_name" -v
+  sleep 10
+done
```

Comment:
Simplified version of the old headless script we were using, this one intentionally requires manually configuration so that users actually know and learn how to configure their own instances of sshnpd